iib/workers/config.py

```diff
@@ -187,14 +202,18 @@ def validate_celery_config(conf, **kwargs):
 
     _validate_iib_org_customizations(conf['iib_organization_customizations'])
 
-    iib_request_logs_dir = conf.get('iib_request_logs_dir')
-    if iib_request_logs_dir:
-        if not os.path.isdir(iib_request_logs_dir):
-            raise ConfigError(
-                f'iib_request_logs_dir, {iib_request_logs_dir}, must exist and be a directory'
-            )
-        if not os.access(iib_request_logs_dir, os.W_OK):
-            raise ConfigError(f'iib_request_logs_dir, {iib_request_logs_dir}, is not writable!')
+    for directory in ('iib_request_logs_dir', 'iib_request_related_bundles_dir'):
+        iib_request_temp_data_dir = conf.get(directory)
+        if iib_request_temp_data_dir:
+            if not os.path.isdir(iib_request_temp_data_dir):
+                raise ConfigError(
+                    f'iib_request_temp_data_dir, {iib_request_temp_data_dir}, '
```

Comment:
`iib_request_temp_data_dir` is only defined within this function. Should the error message refer to the variable `directory` instead? It would make the error easier to diagnose.

Reply:
This still needs to be addressed.

Reply:
oops! apologies